src/drivers/uavcannode/UavcanNode.cpp

```diff
@@ -824,6 +824,27 @@ extern "C" int uavcannode_start(int argc, char *argv[])
 		}
 	}
 
+	// Read the static node ID parameter and use it if no valid node_id from shared memory
+	int32_t cannode_node_id = 0;
+	param_get(param_find("CANNODE_NODE_ID"), &cannode_node_id);
+
+	// Check if the static node ID is in range
+	if (cannode_node_id < 0 || cannode_node_id > 127) {
+		PX4_ERR("Invalid static node ID %d, using dynamic allocation", cannode_node_id);
+		cannode_node_id = 0;
+	}
+
+	// Assign the static node ID if no dynamic allocation is used. Do wen't override a valid node ID from the bootloader?
```

Comment:
This setup will use the dynamic ID from the bootloader, do we want to override it?